.github/scripts/check-tests.mjs

```diff
@@ -17,14 +17,32 @@ const filterAsync = async (asyncPredicate, arr) => {
 	return filterResults.filter(({shouldKeep}) => shouldKeep).map(({item}) => item);
 }
 
+const isAbstractClass = (node) => {
```

Comment:
This should have been a separate PR (without the `fix` in title). This way when cherry-picking the fix into a patch release, we don't have to think about if this change is relevant to a release or not.